README.md

```diff
@@ -1,38 +1,137 @@
+React Apollo unlike many other tools in the React ecosystem requires _no_ complex build setup to get up and running. As long as you have a GraphQL server you can get started building out your application with React immeadiately. React Apollo works out of the box with both [`create-react-app`][] and [React Native][] with a single install and with no extra hassle configuring Babel or other JavaScript tools.
 
-- [Install](#install)
-- [Docs](http://docs.apollostack.com/apollo-client/react.html)
+[`create-react-app`]: https://github.com/facebookincubator/create-react-app
+[React Native]: http://facebook.github.io/react-native
 
-### Documentation
+The Apollo frontend toolchain is built to be:
 
-Documentation for this client can be found [here](http://docs.apollostack.com/apollo-client/react.html);
+1. **Incrementally adoptable**, so that you can drop it into an existing JavaScript app and start using GraphQL for just part of your UI.
+2. **Universally compatible**, so that Apollo works with any build setup, any GraphQL server, and any GraphQL schema.
+2. **Simple to get started with**, you can start loading data right away and learn about advanced features later.
+3. **Inspectable and understandable**, so that you can have great developer tools to understand exactly what is happening in your app.
+4. **Built for interactive apps**, so your users can make changes and see them reflected in the UI immediately.
+4. **Small and flexible**, so you don't get stuff you don't need. The core is under 25kb compressed.
+5. **Community driven**, Apollo is driven by the community and serves a variety of use cases. Everything is planned and developed in the open.
 
-### Local Development
+Get started today on the app you’ve been dreaming of, and let React Apollo take you to the moon!
 
-If you'd like to run a local copy of this package, you can follow these steps:
+## Installation
 
-- Clone this repo locally.
-- In your local `react-apollo` directory: `npm link` then `npm run compile`.
-- In your app's directory: `npm link react-apollo`.
-
-## Install
+It is simple to install React Apollo.
 
 ```bash
 npm install react-apollo --save
 ```
 
-Running tests locally:
+That’s it! You may now use React Apollo in any of your React environments.
+
+For an amazing developer experience you may also install the [Apollo Client Developer tools for Chrome][] which will give you inspectability into your React Apollo data.
+
+[Apollo Client Developer tools for Chrome]: https://chrome.google.com/webstore/detail/apollo-client-developer-t/jdkknkkbebbapilgoeccciglkfbmbnfm
+
+## Usage
 
+To get started you will first want to create an instance of [`ApolloClient`][] and then you will want to provide that client to your React component tree using the [`<ApolloProvider/>`][] component. Finally, we will show you a basic example of connecting your GraphQL data to your React components with the [`graphql()`][] enhancer function.
```

Comment:
I think a lot of the sentences in this PR would be more straight forward to read if instead of saying: "TO X you will want to Y" it just said "Do Y to get X".